src/libs/CardUtils.ts

```diff
@@ -194,6 +195,16 @@ function getCardFeedIcon(cardFeed: string): IconAsset {
     return Illustrations.AmexCompanyCards;
 }
 
+function getDescriptionForPolicyDomainCard(domainName: string) {
```

Comment:
I feel like this method is not really reusable for now, lets just put it into the component